test

```diff
@@ -33,9 +33,13 @@ while [ $# -gt 0 ]; do
             shift 1
             ;;
         "-netgo")
-            TAGS="-tags netgo"
+            TAGS="netgo"
             shift 1
             ;;
+        "-tags")
+            TAGS="$2"
```

Comment:
Not too familiar with the background and Go flags, but is it expected that we simply set values instead of appending to `TAGS` in `TAGS="netgo"` and `TAGS="$2"`?
I am asking cause `TAGS` is now treated as an array (`-tags "${TAGS[@]}`).